testing-internal/src/main/java/com/linecorp/armeria/internal/testing/webapp/WebAppContainerTest.java

```diff
@@ -347,4 +354,27 @@
             }
         }
     }
+
+    @ParameterizedTest
+    @EnumSource(value = SessionProtocol.class, names = { "H1", "H2" })
```

Comment:
nit: Could simply use `@CsvSource({"H1", "H2"})`?

Reply:
That's a good idea. :+1: